Commonalities/documentation/Working/CAMARA-AuthN-AuthZ-Concept.md

```diff
@@ -141,6 +148,97 @@ The authorization code is a temporary code that the client will exchange for an
 
+## Identification of UEs <a name="ueid"></a>
+
+### General Public Subscription Identifier (GPSI) <a name="gpsi"></a>
+
+3GPP defines the GPSI in TS 23.501 [10] clause 5.9.8 and TS 23.003 [12] clause 28.8.
+
+The Generic Public Subscription Identifier (GPSI) was created for addressing a 3GPP subscription in data networks outside of the 3GPP system, so that internal identifiers, such as the Subscription Permanent Identifier (SUPI), were not required to be disclosed to external parties .
+
+As such, 5G Core stores an association between each internal SUPI and one or more GPSIs associated to such SUPI.
+
+The GPSI has two possible formats:
+
+1. An MSISDN (defined in 3GPP TS 23.003 [12] clause 3.3). For example: 1555123456
+2. An External Identifier (defined in 3GPP TS 23.003 [12] clause 19.7.2), which has the format of _username@realm_ as specified in clause 2.1 of IETF RFC 4282 [RFC 4282].
+
+Example: user1AD4@subdomain.example.com
+
+### External Identifier format of the GPSI <a name="extid"></a>
+
+With respect to the External Identifier format of the SUPI, 3GPP TS 23.003 [12] indicates in clause 19.7.2 that the _username_ part contains a _Local Identifier_ as specified in 3GPP TS 23.682 [13] and the _realm_ part contains a _Domain Identifier_as specified in 3GPP TS 23.682 [13], which must be a duly registered Internet domain name.
+
+Observe that the combination of the Local Identifier and the Domain Identifier makes the External Identifier globally unique.
+
+3GPP TS 23.682 [13] specifies the _Domain Identifier_ as a domain that is under the control of a Mobile Network Operator (MNO) and then indicates that the _Domain Identifier_ is used to identify where services provided by the operator network can be accessed. It is possible for an operator to differentiate services by providing different domain identifiers associated to the same subscription.
+
+The _Local Identifier_, also according to 3GPP TS 23.682 [13], is used to derive or obtain the SUPI. It is unique within the applicable domain and managed by the Mobile Network Operator.
+
+Although the concepts of the _Domain Identifier_ and _Local Identifier_ were initially created for Cellular IoT environment, these are generally applicable to any type of user subscription in the mobile network.
+
```

Comment:
After we explain the 2 possible formats of GPSI and the details about the external Identifier format, it would be good to explain, the possible ways on how a developer can get access to these identifiers

Reply:
I guess there are three options for developers to get the GPSI.
a) Delivered by the CSP to the enterprise via offline means
b) by invoking the Nnef_UEID_Get service operation, if the IP address of the UE is known.
c) Received in a response to another operation. A few service operations allow the network to deliver the AF-Specific identifier.

There is already a section in the text describing how an AF can obtain the AF-specific identifier. Perhaps the missing point is the offline delivery means.